docs/sphinx/source/whatsnew/v0.9.6.rst

```diff
@@ -11,7 +11,10 @@ Deprecations
 
 Enhancements
 ~~~~~~~~~~~~
-
+* Added horizon retrieval from PVGIS function in 
+  :py:func:`pvlib.iotools.pvgis.get_pvgis_horizon`. (:issue:`1290`, :pull:`1395`)
+* Added SRTM Digital Elevation Model retrieval from CGIAR in
+  :py:func:`pvlib.iotools.cgiar.download_SRTM`. (:issue:`1290`, :pull:`1395`)
```

Comment:
Also can you please edit the bottom of this file to list the many and various contributors here?  Looking back through the history, here's who I see participation from:
```
- Ben Pierce (:ghuser:`bgpierc`)
- Joseph Palakapilly (:ghuser:`JPalakapillyKWH`)
- Cliff Hansen (:ghuser:`cwhanse`)
- Anton Driesse (:ghuser:`adriesse`)
- Will Holmgren (:ghuser:`wholmgren`)
- Mark Mikofski (:ghuser:`mikofski`)
- Karel De Brabandere (:ghuser:`kdebrab`)
- Josh Stein (:ghuser:`jsstein`)
- Kevin Anderson (:ghuser:`kandersolar`)
```
Hopefully I haven't missed anyone!  This addition seems to have taken a village :)

Reply:
Added all the contributors!

Oh, I had thought that those files in `shading.py` had already been merged for some reason. They were originally written by @JPalakapillyKWH in #758, but were not merged. In that thread, it was suggested that this be [split into 3 tasks](https://github.com/pvlib/pvlib-python/pull/758#issuecomment-905927124). So maybe we should remove these from this PR, both to avoid trying to add too much in one go, and give me some time to write some tests for them :)